recipes/monocle3-cli/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
         - r-base <4
         - r-optparse
         - r-monocle3 <=0.2.0
+        - mscorefonts
```

Comment:
I think a better way is to add `fonts-conda-ecosystem`

Reply:
Do you know if this sorts the problem that we are after? https://github.com/bioconda/bioconda-recipes/pull/16813#issuecomment-520941138 I know mscorefonts does. If you know it does, happy to go for the change :-).

Reply:
`fonts-conda-ecosystem` should be the more general font package that we want to add to r-base or fontconfig in general ...